misaligned.spec.js

```diff
@@ -0,0 +1,45 @@
+const {expect} = require('chai');
+const  print_color_map = require('./misaligned.js');
+// Import the print_color_map function
+// Redirect console.log to a custom function
+const originalLog = console.log;
+const capturedLogs = [];
+
+console.log = function (...args) {
+    capturedLogs.push(args.join(' '));
+    originalLog.apply(console, args);
+};
+
+//console.log(capturedLogs);
+
+ describe('print_color_map Function', () => {
+    it('should have aligned "|" characters', () => {
+        
+        // Call the function to print the color map
+        result = print_color_map();
```

Comment:
Always use let or const... without those, a variable like `result` is shared by all functions in the program. This can lead to unexpected effects in a large program

Reply:
Oh I see. I have updated the code.